pkg/csi/service/common/commonco/k8sorchestrator/k8sorchestrator.go

```diff
@@ -1102,6 +1102,8 @@ func (c *K8sOrchestrator) IsFSSEnabled(ctx context.Context, featureName string)
 						fssVal, common.KubeSystemNamespace, common.WCPCapabilityConfigMapName, err)
 					return false
 				}
+				log.Debugf("Supervisor feature state %q in WCP cluster capabilities is set to %t", featureName,
```

Comment:
Can we use info level log?

Reply:
This part of the code gets executed everytime we call `IsFSSEnabled` for the podVM feature. I have instead info logged the whole WCP cluster capabilities map above which will only run during init.